python/nvstrings/cpp/CMakeLists.txt

```diff
@@ -103,21 +103,25 @@ endif (RMM_INCLUDE AND RMM_LIBRARY)
 find_path(
     NVSTRINGS_INCLUDE "nvstrings"
     HINTS "$ENV{NVSTRINGS_ROOT}/include"
+          "../../../cpp/include" # allow building if nvstrings built locally but not installed
```

Comment:
This is not dependent on nvstrings being built, this is just where the source for the headers lives.

Reply:
Yes but the other three hints are. I didn't want to duplicate the comment...

Reply:
Removed.